### ISSUE TITLE
Lambda Insights Support

### ISSUE THREAD

**Issue by reik-wargaming** [aws/chalice#1752]
Just curious if Chalice has any support to enable Lambda Insights? I think when you enable Lambda Insights using the AWS Console, then "Lambda adds the Lambda Insights extension as a layer to your function, and verifies or attempts to attach the CloudWatchLambdaInsightsExecutionRolePolicy policy to your function’s execution role." (see https://docs.aws.amazon.com/lambda/latest/dg/monitoring-insights.html#monitoring-insights-enabling-console)

I am not sure if the layer somehow can be added manually instead since that would already be possible.

**Comment by jamesls** [comment]
Yeah, it looks like you'd have to manually add the layer and policy if you wanted to do this programmatically: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Lambda-Insights-Getting-Started-cloudformation.html

Right now Chalice doesn't have support for automatically adding that for you.
